apps/markdown-editor/src/MarkdownEditorTopbar.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <oc-topbar variation="secondary" uk-sticky="offset: 60">
+  <oc-topbar variation="secondary" uk-sticky="offset: 60px">
```

Comment:
The units wasn't the problem - uikit use pixels in default in `uk-sticky`. There has been a change in the way we scroll the app - top-bar and app-bar doesn't scroll at all. Only the content of app is scrolling thanks to class uk-overflow-auto. The right fix then is to remove the stickyness completely from that component.

Reply:
Thanks! Will have a look at it.